lisp/mastodon-tl.el

```diff
@@ -483,26 +489,39 @@ message is a link which unhides/hides the main body."
      (mastodon-tl--render-text content)
      (mastodon-tl--media toot))))
 
-(defun mastodon-tl--toot (toot)
-  "Display TOOT content and byline."
+(defun mastodon-tl--insert (toot body author-byline action-byline)
```

Comment:
`mastodon-tl--insert` is too generic  name.   
If this going to be more generic than just inserting toots, we should come up with a good name for this generic "something" and then also rename the `toot` argument and fix the doc comment to not say `toot`.